### PR TITLE
[new release] patricia-tree (0.9.0)

### PR DESCRIPTION
**Pull request by dlesbre**
Patricia Tree data structure in OCaml for maps and sets. Supports generic key-value pairs

- Project page: <a href="https://codex.top/patricia-tree/">https://codex.top/patricia-tree/</a>
- Documentation: <a href="https://codex.top/patricia-tree/">https://codex.top/patricia-tree/</a>

##### CHANGES:

- Initial release of Patricia Tree

### CODE REVIEW

**Comment by raphael-proust** [comment]
Thanks for the contribution

File: packages/patricia-tree/patricia-tree.0.9.0/opam [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+]
+license: "LGPL-2.1-only"
+homepage: "https://codex.top/patricia-tree/"
+doc: "https://codex.top/patricia-tree/"
+bug-reports:
+  "https://github.com/codex-semantics-library/patricia-tree/issues"
+depends: [
+  "ocaml" {>= "4.14"}
+  "zarith" {>= "1.13"}
+  "dune" {>= "2.7" & >= "2.0"}
```
Comment (raphael-proust) [inline packages/patricia-tree/patricia-tree.0.9.0/opam:17]:
```suggestion
  "dune" {>= "2.7"}
```

This should be equivalent but simpler.

Comment (dlesbre) [reply packages/patricia-tree/patricia-tree.0.9.0/opam]: Nicely spotted, I pushed a fix. Not sure why dune outputs these double requirements when generating opam files.

**Comment by mseri** [comment]
Thanks

**Pull request merged**